source/guide_dokuwiki.rst

```diff
@@ -87,14 +87,63 @@ You will need to enter the following information:
   * Allow users to register themselves: your decision
   * Choose under which license your content will be published
 
-At least you have to delete the ``install.php``.
+At least you have to delete or rename the ``install.php``.
 
 Tuning
 ======
 
 For plugins, themes and other stuff go to DokuWiki_.
 
-
+Nice URLs
+=========
+
+To use nice urls like ``https://isabell.uber.space/dokuwiki-on-uberspace`` instead of ``https://isabell.uber.space/doku.php?id=dokuwiki-on-uberspace`` you need to create a ``.htaccess`` file in the root directory of your dokuwiki installation. You can either copy the existing ``.htaccess.dist`` file and start editing it or paste the following configuration text into a new ``.htaccess`` file.
```

Comment:
I've already documented that in [my wiki](https://lar.ven.uber.space/services/dokuwiki#url_rewriting) for your inspiration and suggest also enabling [safe](https://www.dokuwiki.org/config%3Afnencode) URL-encoding to avoid long or corrupt filenames

Reply:
The example in your wiki looks fine, but for here I would suggest to replace 

> between RewriteEngine On and first RewriteRule directive. Finally uncomment all of the following rewrite rules and save changes. 

with showing the three lines (before, new, after) in a codeblock and maybe the new rewrite rules as well in a codeblock